calcom/cal.com
# fix: Add feedback when user installs app [CAL-2144]

## What does this PR do?



This PR adds feedback when a user installs an app when the team dropdown selector is rendered. This also prevents users from clicking the install button multiple times.

This is what the loading state will look like:
<img width="567" alt="CleanShot 2023-07-14 at 22 24 16@2x" src="https://github.com/calcom/cal.com/assets/65426560/ff4e9bfc-0b75-49a0-911a-959fd72448c3">


Fixes # (issue)



## Type of change




## How should this be tested?



- Create a team / org
- Go to the app store
- Click "Install" on the QR code app (used as an example)
- Choose either the test account or team
    - The loading state should appear
    - You should not be able to close the dropdown menu until the install process returns
- Repeat this for the QR code app page

## Mandatory Tasks


## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
